front/src/components/AdminPanel/Panel/InsertQualificationScores/InsertQualificationScores.js

```diff
@@ -0,0 +1,652 @@
+                    {[4, 5, 6].map((i) => (
+                        <input
+                            type="text"
+                            className="smallInput"
+                            key={"i" + i + "-" + competitorId}
+                            id={"i" + i + "-" + competitorId}
+                            onChange={isValidScoreInput}
+                        />
+                    ))}
+                </Row>
+            </div>
+        );
+
+        if (seriesType === 3) return <div key={competitorId}>{inputs_1_3}</div>;
+        else if (seriesType === 6)
+            return (
+                <div key={competitorId}>
+                    {inputs_1_3}
+                    {inputs_4_6}
+                </div>
+            );
+    }
+
+    let checkSendedScores = () => {
+        let inputsCheck = true;
+        for (var i = 0; i < currentParticipantsScores.length; i++) {
+            if (
+                !document.getElementById(
+                    "sum-" + currentParticipantsScores[i]._id
+                ).disabled &
+                !document.getElementById(
+                    "i1-" + currentParticipantsScores[i]._id
+                ).disabled
+            ) {
+                var t = 0;
+                for (var t = 0; t < seriesType; t++) {
+                    if (currentParticipantsScores[i].arrows[t] === "") t++;
+                }
+                if (t < seriesType) inputsCheck = false;
+                if (
+                    currentParticipantsScores[i].score === "" ||
+                    currentParticipantsScores[i].Xs === "" ||
+                    currentParticipantsScores[i].tens === ""
+                ) {
+                    inputsCheck = false;
+                }
+            } else if (
+                document.getElementById(
+                    "sum-" + currentParticipantsScores[i]._id
+                ).disabled
+            ) {
+                for (t = 0; t < seriesType; t++) {
+                    if (currentParticipantsScores[i].arrows[t] === "")
+                        inputsCheck = false;
+                }
+            } else if (
+                currentParticipantsScores[i].score === "" ||
+                currentParticipantsScores[i].Xs === "" ||
+                currentParticipantsScores[i].tens === ""
+            ) {
+                inputsCheck = false;
+            }
+        }
+        console.log(inputsCheck);
+        return inputsCheck;
+    };
+
+    function handleSubmitScores() {
+        if (currentParticipantsScores.length > 0) {
+            if (checkSendedScores() === true) {
+                scores = currentParticipantsScores;
+                for (var i = 0; i < currentParticipantsScores.length; i++) {
+                    var c = 0;
+                    for (var j = 0; j < seriesType; j++) {
+                        if (scores[i].arrows[j] === "") c++;
+                    }
+                    if (c === seriesType) scores[i].arrows = [];
+                }
```

Comment:
You don't use 'i' to anything beside retrieving current element from array - use for ... of, it hides some complexity. Probably same story with internal loop